tests/integration/server.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/python
+
+# Copyright 2016 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import time
+
+import google.cloud.logging
+import google.cloud.monitoring
+import google.cloud.exceptions
+
+from flask import Flask, request, jsonify
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def hello_world():
+    return 'Hello World!'
+
+
+@app.route('/logging', methods=['POST'])
+def _logging():
+    request_data = request.get_json()
+    if request_data is None:
+        raise ErrorResponse('Unable to parse request JSON: '
+                            'did you set the Content-type header?')
+    log_name = request_data.get('log_name')
+    if not log_name:
+        raise ErrorResponse('Please provide log name')
+    token = request_data.get('token')
+    if not token:
+        raise ErrorResponse('Please provide token name')
+
+    _log("log name is {0}, token is {1}".format(log_name, token))
+    _log(token, log_name)
+
+    return 'OK', 200
+
+
+# TODO (nkubala): just as a note, currently the client logging API is broken
```

Comment:
sure. I wayyy overused TODOs in this code, mostly because they're just notes to self as I'm writing the code. before I merge this all (or at least the vast majority) of these will be addressed or converted to issues.